webportal/webapp/test_private_deploy.sh

```diff
@@ -0,0 +1,19 @@
+#/bin/bash
+
+echo "***"
+echo "Building assets..."
+ng build --configuration=test-private
+
+
+# Use distinct test-private bucket so we can modify URLs, etc. per the environment
+# Copy everything over to S3 bucket
+echo "Copying to s3..."
+aws s3 cp --profile sdc ./dist s3://test-private-sdc-webportal-hosting --recursive --only-show-errors
```

Comment:
Did this bucket already get created?

Reply:
> Did this bucket already get created? If so, is it in Terraform somewhere?

I created this bucket manually. I couldn't find anywhere in terraform.

Reply:
Likewise for the other temporary s3 private buckets. Might be worth adding the bucket creation into webportal terraform once we add some of the other pre-reqs.